src/webgl/p5.Geometry.js

```diff
@@ -1910,6 +1917,19 @@ p5.Geometry = class Geometry {
     }
     return this;
   }
+
+  setAttribute(attributeName, data){
+    const size = data.length ? data.length : 1;
+    if (!this.hasOwnProperty(attributeName)){
```

Comment:
It's probably unlikely, but in theory a bit of user data could clash with some of our own properties, e.g. if someone tries to make their own `vertices` property. Does anything else break if we store the user attributes in `this.attribs[attributeName]`? If `RenderBuffer` doesn't handle nested properties like that, we could also always record the initial properties and throw an error if a user tries to set an attribute with a core property name.

Reply:
I will have to add something for this, at least to protect against `attributeName` being `aPosition`, `aNormal` and so on, but we can just concatenate something on the end of the javascript property, i.e. `attributeName` becomes `attributeNameSrc` for the first argument here:
```js
  if(!bufferExists){
    this.immediateMode.buffers.user.push(
      new p5.RenderBuffer(size, attributeName, buff, attributeName, this)
    );
  }  
```

Reply:
That's a great idea that doesn't involve shuffling around any of our other variables! Maybe just leave a comment somewhere that we can't name our builtin attributes with an Src suffix